<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zawodnicyForm</class>
 <widget class="QWidget" name="zawodnicyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zawodnicy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Imię:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nazwisko:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="7">
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>182</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>182</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Pozycja:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_dodaj">
     <property name="text">
      <string>Dodaj</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Drużyna:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButton_usun">
     <property name="text">
      <string>Usuń</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
